packages/graph/migrations/postgres/scripts/environment.ts

```diff
@@ -1,10 +1,11 @@
 import { config } from "dotenv-flow";
+import dotenvExpand from "dotenv-expand";
 
 import path from "path";
 
 export const graphRootDir = path.resolve(__dirname, "../../../");
 
-config({ silent: true, path: graphRootDir });
+dotenvExpand.expand(config({ silent: true, path: graphRootDir }));
```

Comment:
I needed to add this to get `HASH_GRAPH_PG_MIGRATION_URL` to correctly parse the environment variables it contains